src/main/java/core/basesyntax/reportservice/ReportCreatorService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.reportservice;

```

Comment:
fix package structure for all classes
![Screenshot 2022-08-05 at 08 19 35](https://user-images.githubusercontent.com/60258373/183006837-56d4e0e5-8890-4caf-b61b-dc7782514919.png)